packages/nextjs/src/server/keyless-node.ts

```diff
@@ -114,6 +109,32 @@ async function createOrReadKeyless(): Promise<AccountlessApplication | undefined
       flag: 'w',
     },
   );
+};
+
+const unlockFileWriting = () => {
+  const { rmSync } = safeNodeRuntimeFs();
+
+  rmSync(CLERK_LOCK, { force: true, recursive: true });
```

Comment:
I'd suggest adding error handling to the `rm` calls (https://nodejs.org/dist/v18.20.5/docs/api/fs.html#fsaccesspath-mode-callback) e.g. when permissions don't allow you to delete the file. I know it's created with 0777 above but who knows what people might do in between